### DISCUSSION TITLE
Add a Button that can Navigate to Dashboard (for Kiosk Mode)

### DISCUSSION THREAD

**Discussion by victor-vz** [music-assistant/support#2348] [Feature requests and ideas]
For non admins and wall tablets I enabled kiosk mode. In home assistant this deactivates the blue header and the side bar. This cleans up dashboards and ease of use. However I would still like to access Music Assistant as the add on works great. My quick fix right now is to integrate an iframe with the webbrowser (not the cleanest solution).

What I want to accomplish is that if I click on a speaker I navigate to Music Assistant (already possible). I just need a way to get back to the dashboard when I am in Music Assistant with kiosk mode enabled (just a navigate button that is programmable).

![IMG_6871](https://github.com/music-assistant/support/assets/84272274/37deb7c7-6f8a-4bfb-bbf7-0fb4716a083f)

**Comment by Ashleyde42** [comment]
I would like a solution to this.  Plus the Energy dashboard.  How do I get back to mobile main dashboard?

Any further updates on this topic?

**Comment by cannondale0815** [comment]
Not much traction in over a year it seems. Anyone figured this out?

**Comment by sheyman1** [comment]
Not sure if this helps, but on my setup I can click the 3 lines in the top left which then shows the Home Assistant sidebar.
